query/src/interpreters/interpreter_query_log.rs

```diff
@@ -31,7 +32,9 @@ use crate::sessions::QueryContext;
 pub enum LogType {
     Start = 1,
     Finish = 2,
-    Error = 3,
+    ErrorBeforeStart = 3,
```

Comment:
Is there a better name? or add some comment?

Reply:
ErrorExecute means  and `interpreter.exeute()`
ErrorBeforeStart means error before it:  parse, binding, and any other error.

maybe not need to distinguish the two?  @BohuTANG 
or distinguish by `parse_error` and `other_error` sounds more reasonable to me. because for ddl SQLs, some checks may be done in either/both planer or interpreter?

Reply:
ErrorExecute is ok :D

Reply:
One error makes sense

Reply:
done